src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -139,7 +147,11 @@ public void windFlywheel(double rpm) {
     } else {
       DTRPM.setDouble(rpm);
       targetRPM = rpm;
-      flywheelPID.setReference(rpm, CANSparkMax.ControlType.kVelocity);
+      flywheelPID.setReference(
+          rpm,
+          CANSparkMax.ControlType.kVelocity,
+          Constants.Shooter.kMaxISlot,
+          flywheelFF.calculate(rpm / 62.0));
```

Comment:
It was 60 to convert to rps, but it seemed to only be exact at 62 when I was on testing bot
However, I just realized that it was because the 2 shooters have completely different characterizations, so I just changed it back to 60 to work with the competition robot.